sktime/forecasting/dynamic_factor.py

```diff
@@ -0,0 +1,429 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+"""Implements DynamicFactor Model."""
+
+import inspect
+
+import numpy as np
+from statsmodels.tsa.statespace.dynamic_factor import DynamicFactor as _DynamicFactor
+
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+_all_ = ["DynamicFactor"]
+__author__ = ["Ris-Bali"]
+
+_check_soft_dependencies("dynamicfactor", severity="warning")

```

Comment:
there is no such package, at least it is not imported. Also, `statsmodels` is a core dependency, so you can remove this line.